phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -631,7 +664,7 @@ private boolean addIndexesFromPhysicalTable(MetaDataMutationResult result, Long
         if (view.getType() != PTableType.VIEW || view.getViewType() == ViewType.MAPPED) {
             return false;
         }
-        String physicalName = view.getPhysicalName().getString();
+        String physicalName = view.getPhysicalNames().get(0).toString();
```

Comment:
Please undo this change as we still don't allow creating views over multiple tables.